pytorch_toolkit/object_detection/oteod/oteod/evaluation/horizontal_text_detection/horizontal_text_detection.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=R0913
+
+import logging
+import os
+import subprocess
+
+from oteod import MMDETECTION_TOOLS
+from oteod.evaluation.common import collect_ap, evaluate_internal
+
+
+def collect_f1(path):
+    """ Collects precision, recall and f1 score values in log file. """
+    metrics = ['recall', 'precision', 'hmean']
+    content = []
+    result = []
+    with open(path) as read_file:
+        content += [line.split() for line in read_file.readlines()]
```

Comment:
The same question about the necessity to use `.readlines()` here.

Reply:
fixed